l3packages/xparse/xparse.ins

```diff
@@ -42,8 +42,7 @@ and all files in that bundle must be distributed together.
 
 \keepsilent
 
-\generate{\file{xparse-generic.tex}{\from{xparse.dtx}{core}}
-          \file{xparse.sty}{\from{xparse.dtx}{package}}
+\generate{\file{xparse.sty}{\from{xparse.dtx}{package}}
           \file{xparse.ltx}{\from{xparse.dtx}{2ekernel}}}
```

Comment:
what kind of 2ekernel is still produced here? (or is that just temp while the master kernel has different code?)

Reply:
`xparse.ltx` is currently needed to build the 2020-10-01 LaTeX format, so we can't take it out just yet.

Reply:
that's what I thought (hoped :-)) but I couldn't check easily